src/main/java/ca/gc/aafc/collection/api/validation/GeoreferenceAssertionValidator.java

```diff
@@ -7,31 +7,37 @@
 import org.springframework.validation.Validator;
 
 import ca.gc.aafc.collection.api.entities.GeoreferenceAssertion;
+import ca.gc.aafc.collection.api.entities.GeoreferenceAssertion.GeoreferenceVerificationStatus;
 
 @Component
 public class GeoreferenceAssertionValidator implements Validator {
 
-    private MessageSource messageSource;
+  private MessageSource messageSource;
 
-    public GeoreferenceAssertionValidator(MessageSource messageSource) {
-        this.messageSource = messageSource;
-    }
-
-    @Override
-    public boolean supports(Class<?> clazz) {
-        return GeoreferenceAssertion.class.isAssignableFrom(clazz);
-    }
+  public GeoreferenceAssertionValidator(MessageSource messageSource) {
+    this.messageSource = messageSource;
+  }
 
-    @Override
-    public void validate(Object target, Errors errors) {
-        GeoreferenceAssertion georeferenceAssertion = (GeoreferenceAssertion) target;
+  @Override
+  public boolean supports(Class<?> clazz) {
```

Comment:
Overides and should carry the `@NonNull`. like `public boolean supports(@NonNull Class<?> clazz)`

Reply:
Should I modify the CollectingEventValidator to also carry the @NonNull in it's params?

Reply:
I am not sure at this time.